src/sentry/integrations/bitbucket/issues.py

```diff
@@ -99,6 +99,9 @@ def create_issue(self, data, **kwargs):
         if not data.get('repo'):
             raise IntegrationFormError({'repo': ['Repository is required']})
 
+        data['content'] = {'raw': data['description']}
```

Comment:
Do we use any markup (blockquotes, code fences) in our issue bodies? Should we flag this content as markdown?

Reply:
We do use markup, but raw is actually required. I tried to include both raw and markup and realized the markup appears to make no difference. If I just put it as raw, it renders the same (with markdown being recognized): 
<img width="1028" alt="screen shot 2019-01-18 at 10 59 30 am" src="https://user-images.githubusercontent.com/12959365/51407254-35f22880-1b10-11e9-9764-e8c6b180eae7.png">

Reply:
The `markup` field looks like it takes a 'format'. Perhaps that format defaults to markdown?